LICENSE

```diff
@@ -18,3 +18,8 @@ twilio-node:
 
 Project site - https://twilio.github.io/twilio-node/
 License - MIT License, https://github.com/twilio/twilio-node/blob/master/LICENSE
+
```

Comment:
I'd like to bias for not using it for now – I think it's really important that we're all on the same page on how our workshops are used (whether they're on GitHub or Flatdox), so we can design for that user experience.

I'd be happy to set up a Google Hangout so we can further discuss this.